examples/AllTests/FEDemoTest.cpp

```diff
@@ -30,19 +30,20 @@
 #include "CppUTest/TestRegistry.h"
 
 #if CPPUTEST_HAVE_FENV
-#include "CppUTestExt/IEEE754ExceptionsPlugin.h"
+    #include "CppUTestExt/IEEE754ExceptionsPlugin.h"
```

Comment:
These are minor issues. But this one didn't needed to be indented, but we can't avoid that, I assume ?

Reply:
For now, I assume this one cannot be different as it is hard to figure out when to indent the #include and when not to.